compiler/rustc_builtin_macros/src/test.rs

```diff
@@ -225,8 +240,12 @@ pub fn expand_test_or_bench(
                 Ident::new(sym::cfg, attr_sp),
                 vec![attr::mk_nested_word_item(Ident::new(sym::test, attr_sp))],
             )),
-            // #[rustc_test_marker]
-            cx.attribute(cx.meta_word(attr_sp, sym::rustc_test_marker)),
+            // #[rustc_test_marker = "path::to::test"]
```

Comment:
If I'm following the code right, this is more like `#[rustc_test_marker = "name_of_test"]`, right? The fact that currently that's a path is sort of an incidental detail, and we might want to change that in the future for some tests, I suppose.

Reply:
It's more like a `#[rustc_test_marker = "sort_key"]`. The fact that tests are sorted by path is a user-visible detail in stable Rust, so I'm not entirely sure it's fair to call the sort an implementation detail.

Reply:
Yeah, I am not saying we should/could change it, but I think making it clear what the intended use is would be good. I like sort key.